snippets/formatNumber.md

```diff
@@ -0,0 +1,32 @@
+---
+title: formatNumber
+tags: date,math,string,intermediate
+---
+
+Returns a number in readable format.
+
+Use `Number.prototype.toString()` to convert a number to string.
+Use `for` loop to loop through the characters in string. Write to a new string variable one character at a time, adding a comma at every third character. Return the newly created string.
+
+
+```js
+const formatNumber = number => {
```

Comment:
The implementation should use `Number.prototype.toLocaleString` instead and be a one-liner with an implicit return. Also, the description should be updated.

Reply:
Nice, thank you for introducing me to toLocaleString. That makes it so easy :)